algorithms/linfa-ica/src/fast_ica.rs

```diff
@@ -247,6 +247,10 @@ impl<F: Float> PredictInplace<Array2<F>, Array2<F>> for FittedFastIca<F> {
         let xcentered = x - &self.mean.view().insert_axis(Axis(0));
         *y = xcentered.dot(&self.components.t());
     }
+
+    fn num_targets(&self) -> usize {
+        todo!()
```

Comment:
What goes here?

Reply:
just `self.components.nrows()` like in PCA, because the number of columns in the transposed component matrix equals to the number of columns in the results which are the number of targets